website/docs/d/mq_broker.html.markdown

```diff
@@ -0,0 +1,45 @@
+---
+layout: "aws"
+page_title: "AWS: aws_mq_broker"
+sidebar_current: "docs-aws-datasource-mq_broker"
+description: |-
+  Provides a MQ Broker data source.
+---
+
+# Data Source: aws_mq_broker
+
+Provides information about a MQ Broker.
+
+## Example Usage
+
+```hcl
+variable "broker_id" {
+  type    = "string"
+  default = ""
+}
+
+variable "broker_name" {
+  type    = "string"
+  default = ""
+}
+
+data "aws_mq_broker" "by_id" {
+  broker_id = "${var.broker_id}"
+}
+
+data "aws_mq_broker" "by_name" {
+  broker_name = "${var.broker_name}"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `broker_id` - (Optional) The unique id of the mq broker.
+* `broker_name` - (Optional) The unique name of the mq broker.
+
+## Attributes Reference
+
+See the [MQ Broker](/docs/providers/aws/r/mq_broker.html) for details on the returned attributes.
```

Comment:
Nitpick: Should probably change the link text to the below so its a little clearer where its going:

```
[`aws_mq_broker` resource](/docs/providers/aws/r/mq_broker.html)
```